packages/frontend/src/components/table/props/getScalingTableColumns.tsx

```diff
@@ -50,7 +50,11 @@ export function getActiveScalingTvlColumns(stagesEnabled: boolean) {
             tooltip: 'Stage of this Layer 2 based on its features.',
             alignCenter: true as const,
             getValue: (project: ScalingTvlViewEntry) => (
-              <StageCell item={project.stage} />
+              <StageCell
+                item={
```

Comment:
Don't we want to show "under review" in the table too?

Reply:
yeah, we will show it